cluster/manifests/skipper/deployment.yaml

```diff
@@ -208,40 +220,50 @@ spec:
           readOnlyRootFilesystem: true
           runAsNonRoot: true
           runAsUser: 1000
-{{ if and (eq .ConfigItems.skipper_local_tokeninfo "bridge") (eq .ConfigItems.enable_apimonitoring "true")}}
-        volumeMounts:
-          - name: routes
-            mountPath: /etc/routes
-          - name: filters
-            mountPath: /etc/config/default-filters
-{{ else if eq .ConfigItems.skipper_local_tokeninfo "bridge"}}
+{{ if or (eq .Cluster.ConfigItems.skipper_local_tokeninfo "bridge") (eq .Cluster.ConfigItems.enable_apimonitoring "true") (eq .Cluster.ConfigItems.skipper_oauth2_ui_login "true")}}
```

Comment:
Same here, I think it'd work fine without the `if`.

Reply:
will try

Reply:
fixed